### PR TITLE
Consolidate dev server into admin server. Organize admin folder

### PR DESCRIPTION
**Pull request by breck7**
This merges devServer into adminServer. This just gives us 1 less process to worry about and restart. The routes are only enabled on dev. Also just cleaned up and normalized the code in the `admin` folder a bit.

### CODE REVIEW

**Review by MarcelGerber — APPROVED** [review]
Except for these few comments, this looks good to me.

**Comment by MarcelGerber** [comment]
I'm sorry, looks like I just created new conflicts for you by merging that other PR 😶

File: adminSite/server/bakedSiteRouter.tsx [hunk 1]
```diff
@@ -22,61 +22,64 @@ import {
...
-import { covidCountryProfileRootPath } from "./covid/CovidConstants"
-import { bakeCovidChartAndVariableMeta } from "./bakeCovidChartAndVariableMeta"
-import { chartExplorerRedirectsBySlug } from "./bakeCovidExplorerRedirects"
+import { covidCountryProfileRootPath } from "../../site/server/covid/CovidConstants"
+import { bakeCovidChartAndVariableMeta } from "../../site/server/bakeCovidChartAndVariableMeta"
+import { chartExplorerRedirectsBySlug } from "../../site/server/bakeCovidExplorerRedirects"
 import { countryProfileSpecs } from "site/client/CountryProfileConstants"
 
-const devServer = express()
+const bakedSiteRouter = Router()
```
[the remark this answers is not in the document]
Comment (breck7) [reply adminSite/server/bakedSiteRouter.tsx]: Good point. I'm not crazy about the name either. The one problem with `devServer` is that we already have `devServer` running on port `8090` for webPack (I guess that is just a webpack name: https://webpack.js.org/configuration/dev-server/). I'll go with "mockSite" for now.

**Comment by breck7** [comment]
> I'm sorry, looks like I just created new conflicts for you by merging that other PR 😶

Np! Very minor for me to merge. The big one on the other hand I need to get to :)

**Pull request merged**